src/pages/pass-finalisation/components/pass-certificate-number/pass-certificate-number.constants.ts

```diff
@@ -0,0 +1,2 @@
+
+export const PASS_CERTIFICATE_NUMBER_CTRL = 'passCertificateNumberCtrl';
```

Comment:
Could you use the static fieldname in the component class instead of creating a duplicate of the constant 

`static readonly fieldName: string = 'passCertificateNumberCtrl';`

Reply:
Would like a unit test to prove that is invalid is displayed for the Pass certificate.

Reply:
I changed the static fieldName to the constant so that there is one source for all of them